src/OpenSage.Game/Game.cs

```diff
@@ -853,6 +853,30 @@ private void LocalLogicTick(IEnumerable<InputMessage> messages)
             // We pass RenderTime to Scene2D so that the UI remains responsive even when the game is paused.
             Scene2D.LocalLogicTick(RenderTime, Scene3D?.LocalPlayer);
             Scene3D?.LocalLogicTick(MapTime, tickT);
+
+            // TODO: do this properly (this is a hack to call StartMultiplayerGame on the correct thread)
+            if (SkirmishManager.SkirmishGame?.ReadyToStart == true)
```

Comment:
I would do just this:

```suggestion
            if (SkirmishManager.SkirmishGame?.ReadyToStart)
```